meta-luneos/recipes-support/waydroid/waydroid-data.bb

```diff
@@ -15,15 +15,15 @@ COMPATIBLE_MACHINE:aarch64 = "(.*)"
 
 WAYDROID_ARCH:x86-64 = "waydroid_x86_64"
 WAYDROID_ARCH:aarch64 = "waydroid_arm64"
-WAYDROID_SYSTEM_IMAGE = "lineage-17.1-20211021-VANILLA-${WAYDROID_ARCH}-system.zip"
-WAYDROID_VENDOR_IMAGE = "lineage-17.1-20211021-MAINLINE-${WAYDROID_ARCH}-vendor.zip"
-WAYDROID_VENDOR_IMAGE:halium = "lineage-17.1-20211021-HALIUM_10-${WAYDROID_ARCH}-vendor.zip"
+WAYDROID_SYSTEM_IMAGE = "lineage-18.1-20231028-VANILLA-${WAYDROID_ARCH}-system.zip" 
+WAYDROID_VENDOR_IMAGE = "lineage-18.1-20231028-MAINLINE-${WAYDROID_ARCH}-vendor.zip" 
+WAYDROID_VENDOR_IMAGE:halium = "lineage-16.0-20231028-HALIUM_9-${WAYDROID_ARCH}-vendor.zip"
```

Comment:
why do you downgrade the halium image ? note that it will therefore not match the gbinder API level

Reply:
I thought we need HALIUM_9 for our Halium 9 targets? Or the naming is misleading? Anyway I don't think we have any Halium target that runs it currently.

Reply:
for waydroid I don't think it matters so much, as it doesn't use specific kernel drivers. Worth testing, there's a chance that halium 11 would work there.

Reply:
OK updated